try{
    let a=10
    let b=0
    console.log(a/b)
}
catch(ArithmaticException){
    console.log(ArithmaticException)

}